pom.xml

```diff
@@ -30,7 +30,7 @@
 
   <groupId>org.terracotta.internal</groupId>
   <artifactId>tc-config-parent</artifactId>
-  <version>10.7-SNAPSHOT</version>
+  <version>10.9-SNAPSHOT</version>
```

Comment:
10.7 to 10.9?  Was the old one incorrect?

Reply:
🤷

Reply:
why 10.9?  this thing just exists for conversion tooling.